edx/edx-app-ios
# LEARNER-7686, Add more video playback speed options

### Description

[LEARNER-7686](https://openedx.atlassian.net/browse/LEARNER-7686)

In this PR I added more options for video playback speed. 

### How to test this PR
- Login into the app.
- Go inside any course.
- Open the video tab and start any video.
- Tap on the setting button on video player.
- Tap on video speed, you should have options, 0.25x, 0.5x, 0.75x, 1.0x, 1.25x, 1.5x, 1.75x, 2.0x.

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-14 at 13 01 56](https://user-images.githubusercontent.com/4454296/79200660-3632be00-7e50-11ea-8b21-f37750efc364.png)
